### ISSUE TITLE
Compile the files into a neat executable

### ISSUE THREAD

**Issue by R0merol** [R0merol/Number-Game-v0.1#9]
I'm pretty sure there's a way to do it. Make it so that it looks just like any other game

**Comment by R0merol** [comment]
I found this tutorial on how to 'compile' it, although it's more like installing it? Idk. Anyway, [here's the link to that video](https://www.youtube.com/watch?v=lTxaran0Cig&list=PLYTA5hneYCV2l6dTfA9yJtik7g7_1sCd8&index=1)

**Comment by Joshix-1** [comment]
It would be great to package it https://packaging.python.org/en/latest/tutorials/packaging-projects/ and make it installable with pip. (Which then could provide an executable)

**Comment by R0merol** [comment]
I've made an executable before with pygame and had to format the file directory like [this project](https://github.com/R0merol/Bomber-Plane-Simulator) so that it compiles neatly. Also, the executable should be called Number-Game.exe. And this will mean a lot of refactoring is gonna be required.

**Comment by colderinit** [comment]
Is there any progress or wish for this issue to retain relevance? @R0merol 
I may be able to help with this!

**Comment by R0merol** [comment]
Yes, I'd like this issue to be completed. But first, I'll have to refactor the code to make it easier to compile
